### ISSUE TITLE
Resource type metadata element

### ISSUE THREAD

**Issue by kitchenprinzessin3880** [pangaea-data-publisher/fuji#117]
Suggestion to include resource type as one of the core properties of F2, instead of data content descriptor.

**Comment by huberrob** [comment]
Agree, this selection would fit with the table 2 of to Fenner (2019) https://www.nature.com/articles/s41597-019-0031-8/tables/2. According to this table only 'version' is not included in all investigated metadata formats. 
Fenner further states that _"Persistent identifiers for datasets must support multiple levels of granularity to support both the citation of a specific version and/or individual dataset, as well the citation of an unspecified version of a dataset and/or a collection of primary data."_
This is probably acknowledging that for some datasets no other version exists. In this case the necessary specificity is always ensured without explicitly stating a version number.

In summary, this justifies the following selection of core citation metadata: 
partial_elements = ['creator', 'title', 'object_identifier', 'publication_date','publisher','object_type']

But I would still prefer to leave object type check in FsF-R1-01MD. One reason is that there we also check the validity (in addition to presence/absence) of a given type against a controlled list derived from DC types, DataCite types and schema.org_context.

**Comment by huberrob** [comment]
See also #107
